packages/features/auth/lib/next-auth-options.ts

```diff
@@ -744,8 +747,8 @@ export const AUTH_OPTIONS: AuthOptions = {
             existingUserWithEmail.identityProvider === IdentityProvider.CAL &&
             (idP === IdentityProvider.GOOGLE || idP === IdentityProvider.SAML)
           ) {
-            await prisma.user.update({
-              where: { email: existingUserWithEmail.email },
+            await prisma.user.updateMany({
+              where: { email: existingUserWithEmail.email, linkedBy: null },
```

Comment:
As discussed, we will default to the parent user profile for authentication processes.